fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -2593,9 +2593,9 @@
 | `defaultValue=(string 'false')`| (changed)| `"false"` |
 | `defaultValue=(string 'on')`| (changed)| `"on"` |
 | `defaultValue=(string 'off')`| (changed)| `"off"` |
-| `defaultValue=(symbol)`| (initial, ssr warning)| `<empty string>` |
-| `defaultValue=(function)`| (initial, ssr warning)| `<empty string>` |
-| `defaultValue=(null)`| (initial, ssr warning)| `<empty string>` |
```

Comment:
Looked at this one. It's because old SSR always assigned this prop internally and so it went through the `value` validation which complains about `null`. Seems fine that it doesn't do that now.